sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -140,19 +140,24 @@ To find the resource ID:
 1. In the resulting JSON, copy the value of the `id` property.
 
 ```C# Snippet:QueryResource
-string resourceId = "/subscriptions/<subscription_id>/resourceGroups/<resource_group_name>/providers/<resource_provider>/<resource>";
-var client = new LogsQueryClient(new DefaultAzureCredential());
-
-Response<LogsQueryResult> result = await client.QueryResourceAsync(
+string resourceId = "<resourceId>";
```

Comment:
Why did we remove the part that shows how to construct the client and gives a more detailed resourceId example?

Reply:
Restored